scripts/patch_computed_only.sh

```diff
@@ -32,3 +31,10 @@ for f in $FILES; do
     replace="s/names.AttrTagsAll:\s+tftags.TagsAttributeComputedOnly/names.AttrTagsAll: tftags.TagsAttribute/g"
     perl -i -p -e "$replace" "upstream/$f"
 done
+
+# Add the tags fixes to the existing commit that manages the tags_all attribute.
+cd upstream
+git add .
+tags_all_commit=$(git log --pretty=format:"%h" --grep="Apply tags patches" -n 1)
+git commit --fixup="$tags_all_commit"
```

Comment:
This allows us to add the changes to the existing patch instead of creating new patches for each individual resource.